csi/deployment_util.go

```diff
@@ -11,9 +12,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	clientset "k8s.io/client-go/kubernetes"
 
+	"github.com/rancher/longhorn-manager/types"
+
 	longhornclient "github.com/rancher/longhorn-manager/client"
 )
 
+const (
+	maxRetryCountForMountPropagationCheck = 10
+	durationSleepForMountPropagationCheck = 10 * time.Second
```

Comment:
The deployer starts after manager starts, so the check should be done fairly quick.

Let's try the interval of 5 seconds.